### PR TITLE
fix(react-avatar): imgRef prop added

### PR DESCRIPTION
**Pull request by danielsimao**
Closes [#476 ](https://github.com/nextui-org/nextui/issues/476)

## 📝 Description

Finalise the development from https://github.com/nextui-org/nextui/pull/475. The PR owner seems to not respond, so just finished his pending changes.

## ⛳️ Current behavior (updates)

> Please describe the current behavior that you are modifying

## 🚀 New behavior

> Please describe the behavior or changes this PR adds

## 💣 Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: packages/react/src/utils/dom.ts [hunk 1]
```diff
@@ -131,7 +131,7 @@ export function createFocusableRef<T extends HTMLElement = HTMLElement>(
 }
 
 export function useDOMRef<T extends HTMLElement = HTMLElement>(
-  ref: RefObject<T | null> | Ref<T | null>
+  ref?: RefObject<T | null> | Ref<T | null>
```
Comment (danielsimao) [inline packages/react/src/utils/dom.ts:134]: Can this be optional @jrgarciadev? I think it could be because of optional refs passed as props.

**Comment by jrgarciadev** [comment]
Hey @danielsimao thanks a lot for sending this PR 🙏🏻, could you please fix the conflicts and format the code using the new eslint rules?

**Comment by danielsimao** [comment]
@jrgarciadev ready!

**Review by jrgarciadev — APPROVED** [review]

**Comment by jrgarciadev** [comment]
Huge thanks! @danielsimao 🙌🏻

**Pull request merged**